catroid/src/main/java/org/catrobat/catroid/utils/FormatNumberUtil.kt

```diff
@@ -20,26 +20,26 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+package org.catrobat.catroid.utils
 
-package org.catrobat.catroid.utils;
+import java.math.BigDecimal
 
-import java.math.BigDecimal;
+class FormatNumberUtil private constructor() {
```

Comment:
using object FormatNumberUtil instead of class FormatNumberUtil private constructor() would reduce code... then you could just declare method inside of it with @JvmStatic before. No need for init throw assertionerror and companion object